src/zc/buildout/buildout.py

```diff
@@ -1237,6 +1237,10 @@ def _initialize(self):
         if name == 'buildout':
             return # buildout section can never be a part
 
+        for dname in self.get('<part-dependencies>', '').strip().split():
```

Comment:
> `.strip().split()` is equivalent to `.split()`.

TIL!

In 1.5 decades of Python I've been expecting empty strings at the ends of the list in case of unstripped whitespace...